message-types.js

```diff
@@ -3,45 +3,77 @@ if(typeof(window) != "undefined") {
 } else {
 }
 
-exports.EMAIL_REGISTER = "register"; // A new email account has been created
-exports.EMAIL_SEND = "send"; // An email has been sent
+exports.EMAIL_REGISTER = "register"; // Create a new email account
```

Comment:
General note for the whole file -- it's fine to have present and past tense be the difference between inbound and outbound (rather than something more explicit like _IN and _OUT) but we should probably add a comment at the top of the file to make it very clear which is which.

The inline comments do help clarify this, although not explicitly.